config/filter.format.basic_html.yml

```diff
@@ -37,12 +37,6 @@ filters:
     status: true
     weight: 9
     settings: {  }
-  filter_image_lazy_load:
-    id: filter_image_lazy_load
-    provider: filter
-    status: true
-    weight: 15
-    settings: {  }
```

Comment:
I'm not sure where these came from, but there's not an associated plugin with these so keeping them in causes Drush to complain when it's importing the settings. So... I just got rid of these. Seems to work? 🤷🏻‍♂️

Reply:
The default format for **Body** and other long text fields is `basic`. I think I as playing around in there. Totally fine to remove it.